server/feeders/restaurantfeeder.js

```diff
@@ -132,28 +132,39 @@ var feedOrionRestaurants = function() {
 
     var attr = {
       'type': 'Restaurant',
-      'id': utils.fixedEncodeURIComponent(rname),
+      'id': utils.asciiEncode(utils.stripForbiddenChars(rname)),
       'address': {
-        'type': 'postalAddress'
+        'type': 'PostalAddress',
+        'value': {}
+      },
+      'name': {
+        'value': utils.fixedEncodeURIComponent(rname)
+      },
+      'department': {
+        'value': utils.randomElement(organization)
       },
-      'department': utils.randomElement(organization),
       'capacity': {
         'type': 'PropertyValue',
-        'name': 'capacity',
         'value': utils.randomElement(capacity)
       },
-      'aggregateRating': {},
+      'aggregateRating': {
+        'type': 'AggregateRating',
+        'value': {}
```

Comment:
If I do not specify it, it returns an `undefined` error when doing `attr.aggregateRating.value.ratingValue`. Maybe null could work. (I'll have a look)